conf/config.go

```diff
@@ -95,14 +95,12 @@ type DBusConfig struct {
 
 func NewMenderConfig() *MenderConfig {
 	return &MenderConfig{
-		MenderConfigFromFile: MenderConfigFromFile{
-			DeviceTypeFile: DefaultDeviceTypeFile,
```

Comment:
The reason this is removed is because we cannot provide a static default, we have to consult the `runOptions.dataStore` path, correct? If so, I think we should also remove the `DefaultDeviceTypeFile` constant.

Reply:
That's right, but DefaultDeviceTypeFile is still used in cli/setup.go.
Then maybe I should make some changes here then?
```
// Make sure devicetypefile and serverURL is set
	if config.DeviceTypeFile == "" {
		// Default devicetype file as defined in device.go
		config.DeviceTypeFile = conf.DefaultDeviceTypeFile
	}

```

Reply:
Oh, right! Hmm, I think it would not be appropriate to use the same construct here, as `setup` is supposed to be run completely from scratch, with no prior configuration.

What I suggest is this: *Move* the `DefaultDeviceTypeFile` to `setup.go` instead, so that it is confined there, and change the code to use that reference instead. Does that make sense?

Reply:
Good idea, i'll do that!